src/analysis/special_functions/integrals.lean

```diff
@@ -5,7 +5,6 @@ Authors: Benjamin Davidson
 -/
 import measure_theory.integral.interval_integral
 import analysis.special_functions.trigonometric.arctan_deriv
-import measure_theory.integral.integral_eq_improper
```

Comment:
(This import is not needed – it's left over from an early draft of my previous PR #14147. I should have taken it out but I forgot.)